### ISSUE TITLE
[Accessibility]WAI-ARIA landmarks are missing in login,Settings html pages.

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/wok#82]
The page must provide a link to skip directly to the main page content as the first link on each page or use WAI-ARIA landmarks.

The page must provide a link at the top as first link in the page to skip directly to the main page content i.e first accordion section in the page. 
we need to use WAI-ARIA landmarks and region attributes.
